tests/test_main.py

```diff
@@ -18,8 +18,8 @@
 )
 def test_count_occurrences(phrase, letter, count):
     assert count_occurrences(phrase, letter) == count, (
-        f"Function 'count_occurrences' should return {count}, "
-        f"when 'phrase'='{phrase}' and 'letter'='{letter}'"
```

Comment:
Rollback this changes, please

Reply:
I had trouble with version 2.7 of Python on my PC, so I changed the test not to use f, as it was recommended on the internet. I have uploaded the 3.7 version of Python, pycharm of 2018.3 (as it is what my PC can follow). I do not know if 3.7 is okay with f. Should I do it, or can I live with the changes? Thank you!

Reply:
I have done, all works. please check, thanks!